devel/geoipupdate/Portfile

```
@@ -1,37 +1,138 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
-PortSystem      1.0
-PortGroup       github 1.0
-PortGroup       legacysupport 1.0
+PortSystem          1.0
+PortGroup           golang 1.0
 
-github.setup    maxmind geoipupdate 3.1.1 v
-revision        1
-categories      devel net
-maintainers     nomaintainer
-description     Maxmind geoip databse updater
-license         GPL-2+
+go.setup            github.com/maxmind/geoipupdate 6.0.0 v
+revision            0
+categories          devel net
+platforms           darwin
+maintainers         nomaintainer
+description         Maxmind geoip databse updater
+license             {apache mit}
 
 long_description    {*}${description}
+homepage            http://dev.maxmind.com/geoip/geoipupdate/
```

Please make all URLs HTTPS if supported:

```suggestion
homepage            https://dev.maxmind.com/geoip/geoipupdate/
```

---

This link now forwards to another page - https://dev.maxmind.com/geoip/updating-databases. Should I update the Portfile to use the redirected link? Or, should I completely remove that line since the golang PortGroup already sets `Homepage` to the GitHub page?

---

I think lets keep the link as is, but HTTPS, letting it redirect.  The page it forwards to still has information pertaining to the program.

---

Thank you. I updated the line to use HTTPS.